middleware/auth.ts

```diff
@@ -4,6 +4,11 @@ export default defineNuxtRouteMiddleware((to) => {
   if (to.path === '/signin/callback')
     return
 
+  if (isHydrated.value) {
+    if (to.path === '/')
+      return navigateTo('/home')
+  }
+
```

Comment:
Is this needed? Maybe the logo could go directly to /home?

Reply:
I think the logo can go directly to `/home`, currently just follow the original design